src/main.rs

```diff
@@ -257,29 +268,46 @@ async fn main() -> anyhow::Result<()> {
                             .await?;
                     }
                 }
-                KeyCode::Up if app.ui_state.time_frame_menu_state.active => {
-                    app.ui_state.time_frame_menu_state.select_prev()?;
+                KeyCode::Up => {
+                    if app.ui_state.time_frame_menu_state.active {
+                        app.ui_state.time_frame_menu_state.select_prev()?;
+                    } else if app.ui_state.indicator_menu_state.active {
+                        app.ui_state.indicator_menu_state.select_prev()?;
+                    }
                 }
-                KeyCode::Down if app.ui_state.time_frame_menu_state.active => {
-                    app.ui_state.time_frame_menu_state.select_next()?;
+                KeyCode::Down => {
+                    if app.ui_state.time_frame_menu_state.active {
+                        app.ui_state.time_frame_menu_state.select_next()?;
+                    } else if app.ui_state.indicator_menu_state.active {
+                        app.ui_state.indicator_menu_state.select_next()?;
+                    }
                 }
                 KeyCode::Char(c) if app.ui_state.stock_symbol_input_state.active => {
                     app.ui_state
                         .stock_symbol_input_state
                         .value
                         .push(c.to_ascii_uppercase());
                 }
-                KeyCode::Char(_) if app.ui_state.time_frame_menu_state.active => {}
```

Comment:
I think we can allow this, I find it quite handy.

Reply:
Actually I plan to make it such that the user can type a key sequence like this:

[T] [5] [D] [Enter]